app/views/coronavirus_landing_page/_page_header.html.erb

```diff
@@ -78,7 +78,14 @@
           <div class="govuk-grid-column-one-third">
             <% announcement_text = capture do %>
               <% if announcement["href"].present? %>
-                <a href="<%= announcement["href"] %>" class="govuk-link covid__page-header-link">
+                <a
+                  href="<%= announcement["href"] %>"
+                  class="govuk-link covid__page-header-link"
+                  data-module="track-click"
+                  data-track-category="pageElementInteraction"
+                  data-track-action="Announcements"
+                  data-track-label="<%= announcement["href"] %>"
+                >
```

Comment:
Have you tested that this works? I think the `data-module="track-click"` might need to be on a parent element of the thing being tracked rather than on the element itself.

Reply:
Yes - this works, afaik this should be on the link. Can you point to any docs that say otherwise?

Reply:
Think this looks good?

```
analytics_debug.js:25 Running command: 
ga("send", {
hitType: "event", 
eventCategory: "pageElementInteraction", 
eventAction: "Announcements", 
eventLabel: "/government/publications/full-guidance-on-staying-at-home-and-away-from-others", 
dimension15: "200", dimension16: "unknown", 
dimension95: "1410835217.1585741689", dimension11: "2", dimension2: "coronavirus_landing_page", 
dimension3: "other", dimension4: "774cee22-d896-44c1-a611-e3109cce8eae", 
dimension12: "not withdrawn", dimension17: "coronavirus_landing_page", dimension20: "Collections", 
dimension30: "none", dimension32: "Taxon Page", dimension56: "other", dimension57: "other", 
dimension58: "other", dimension59: "other", dimension39: "false", dimension89: "collections-publisher", dimension26: "0", dimension27: "0", dimension23: "unknown", transport: "beacon"})
analytics_debug.js:25 Sending hit with transport method "beacon"

```

Reply:
I use a plugin called Omnibug - it shows what events have been triggered. Here is an example for the announcement stuff

![Screenshot 2020-04-01 at 13 24 09](https://user-images.githubusercontent.com/4599889/78136840-2fea1e00-741c-11ea-9b68-f7bd0f772494.png)